src/lib/viewers/media/DashViewer.js

```diff
@@ -191,25 +194,21 @@ class DashViewer extends VideoBaseViewer {
     }
 
     /**
-     * Handler for hd video
+     * Given a videoId (e.g. hd video id), enables the track with that video ID
+     * while maintaining the SAME AUDIO as the active track.
      *
      * @private
+     * @param {number} videoId - the id of the video used in the variant (provided by Shaka)
```

Comment:
Nit: Capitalize `The` in `The id of the video`

Reply:
Done